### PR TITLE
Simmy and .Net versions upgrade

### PR DESCRIPTION
**Pull request by vany0114**
* updates Simmy to the latest version
* refactors monkey policies to use the new syntax API
* updates project to use .net core 3.1

### CODE REVIEW

File: SimmyDemo_WebApi/Chaos/SimmyExtensions.cs [hunk 1]
```diff
@@ -34,19 +36,18 @@ public static IPolicyRegistry<string> AddChaosInjectors(this IPolicyRegistry<str
...
-                                GetEnabled))
-                        ;
+                            .WrapAsync(MonkeyPolicy.InjectExceptionAsync(with =>
+                                with.Fault<HttpResponseMessage>(GetException)
+                                    .InjectionRate(GetInjectionRate)
+                                    .EnabledWhen(GetEnabled)))
+                            .WrapAsync(MonkeyPolicy.InjectResultAsync<HttpResponseMessage>(with =>
+                                with.Result(GetHttpResponseMessage)
+                                    .InjectionRate(GetInjectionRate)
+                                    .EnabledWhen(GetEnabled)))
```
Comment (reisenberger) [inline SimmyDemo_WebApi/Chaos/SimmyExtensions.cs]: This used `GetHttpResponseEnabled` in the deleted code. Should it still?

**Review by reisenberger — COMMENTED** [review]
Thanks for picking this up @vany0114 !  

Can we also update the `CHANGELOG.md`? (give this a new version number and describe the changes)

File: SimmyDemo_WebApi/Chaos/SimmyExtensions.cs (hunk 1 above)
Comment (vany0114) [reply SimmyDemo_WebApi/Chaos/SimmyExtensions.cs]: OMG! Good Catch Dylan, it was a copy/paste mistake 😅

**Comment by vany0114** [comment]
> Thanks for picking this up @vany0114 !
> 
> Can we also update the `CHANGELOG.md`? (give this a new version number and describe the changes)

Done!

**Comment by reisenberger** [comment]
Looks good to me!

**Pull request merged**